tensorflow_datasets/core/registered.py

```diff
@@ -106,6 +107,14 @@
     version=r"[0-9]+\.[0-9]+\.[0-9]+",
 ))
 
+_SKIP_REGISTRATION = False
+
+@contextmanager
+def skip_registration():
+  global _SKIP_REGISTRATION
```

Comment:
CAPITALIZE means constant. Here you modify the variable so should be lower_case

Reply:
Okay